code/design_pattern/src/policy_based_design/readme.md

```diff
@@ -0,0 +1,13 @@
+# Policy Based Design
```

Comment:
Follow our [documentation guide](https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md), please.

Reply:
Do I really need to search for images and do everything written in that guide?

Reply:
@n-is Well an image isn't necessary (You can replace with an explanation), but the rest of it is. We have a agreed-upon style, please follow it!

Reply:
@arnavb Thanks for pointing that out. I have followed the guide and removed first person.